components/file-upload/src/index.js

```diff
@@ -15,7 +15,7 @@ import {
   MEDIA_QUERIES,
 } from '@govuk-react/constants';
 
-const Input = glamorous.input({
+const GInput = glamorous.input({
```

Comment:
I spoke to Alasdair about this and he told me this convention had been put in place whenever there were multiple elements with the same name but also required a reference to Glamorous - which means the change I made here is not really required.